README.md

```diff
@@ -93,7 +93,17 @@ According to the [CPC Charter](https://github.com/openjs-foundation/cross-projec
 - Myles Borins (@MylesBorins)
 - Sendil Kumar (@sendilkumarn)
 
-<!-- ### Observers -->
+### Observers
+
+Anyone who participates in an OpenJS Foundation project can be an Observer. 
```

Comment:
I was under the impression that we wanted to go even more liberal with this...

```suggestion
Anyone can be an Observer.
```

Reply:
+1

Reply:
+1